core/async/src/tokio/runtime_handle.rs

```diff
@@ -0,0 +1,92 @@
+use std::sync::Arc;
+
+use tokio::sync::mpsc;
+
+use crate::futures::{DelayedActionRunner, FutureSpawner};
+use crate::messaging::Actor;
+use crate::tokio::sender::TokioSender;
+
+pub struct TokioRuntimeHandle<A> {
+    sender: TokioSender<A>,
+    runtime: Arc<tokio::runtime::Runtime>,
```

Comment:
Might make more sense to have `tokio::Handle` here instead of `Arc<tokio::Runtime>` and keep just one instance of runtime.